### PR TITLE
modifing the footer content according the updates on HYF original website

### PR DESCRIPTION
**Pull request by Moamin-AbuEwaida**
I modified the footer's content as the new updated content to HYF website, also added the meta link in the head of the program, volunteers and about pages to show the footer's SM icons

### CODE REVIEW

**Comment by danielhalasz** [comment]
@Moamin-AbuEwaida it looks good, thanks for updating it. 1 observation: do you maybe want to use the same text color and font that is in the original footer?

**Comment by danielhalasz** [comment]
h3:   color: rgb(28, 34, 148);
 p:     font-family: "Space Mono", monospace;

**Comment by Moamin-AbuEwaida** [comment]
> h3: color: rgb(28, 34, 148);
> p: font-family: "Space Mono", monospace;

i'll do that now

**Review by bringmetheaxx — APPROVED** [review]
good!

**Comment by danielhalasz** [comment]
@Moamin-AbuEwaida thx, there seems to be something happening to the heading now..it is not centered

File: css/styles.css [hunk 1]
```diff
@@ -923,20 +943,21 @@ header {
 .footer h3 {
   margin: 20px auto;
   font-family: "work sans", "sans";
-  color: rgb(74, 80, 187);
+  color:  rgb(28, 34, 148);
   text-transform: uppercase;
-  font-size: 1.5rem;
+  font-size: 1.5rem;"Space Mono", monospace
```
Comment (danielhalasz) [inline css/styles.css:948]: this is causing the problem

**Review by danielhalasz — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 1 above)
Comment (Moamin-AbuEwaida) [reply css/styles.css]: yeah yeah just saw it and gonna do it :+1: 
thanks

**Review by danielhalasz — APPROVED** [review]
nice!

**Pull request merged**